autovacuum_mail_message/README.rst

```diff
@@ -0,0 +1,66 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
```

Comment:
Use the new url from [template](https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst)
```rst
.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
   :target: https://www.gnu.org/licenses/agpl
   :alt: License: AGPL-3
```